lib/board.rb

```diff
@@ -0,0 +1,58 @@
+require 'pry'
+class Board
+  attr_accessor :board
```

Comment:
Why does this need to be accessible? Could it only be readable?

Reply:
@sssunday minimax is using cells for recursion and it needs write access as well